static/src/javascripts/projects/common/modules/identity/modules/loadEnhancers.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { _ as robust } from 'lib/robust';
+import fastdom from 'lib/fastdom-promise';
+
+const loadEnhancers = (loaders: Array<any>): void => {
```

Comment:
this could perhaps be a global lib / part of robust / (maybe theres already a similar lib? it's so tiny i didn't check first)

Reply:
If you can come up with a better name and a usecase outside of this module I'm very happy to move this into `lib/robust`.

Maybe you could also think about the types, that these loaders pass around?

PS: there is another `any` on line 6. Would be ✨ if you could be more specific with that. See the other comment.